packages/twenty-server/src/core/quick-actions/intelligence.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@nestjs/common';
+
+import axios from 'axios';
+
+import { EnvironmentService } from 'src/integrations/environment/environment.service';
+
+@Injectable()
+export class IntelligenceService {
+  constructor(private readonly environmentService: EnvironmentService) {}
+
+  async enrichCompany(domainName: string): Promise<any> {
+    const enrichedCompany = await axios.get(
+      `https://companies.twenty.com/${domainName}`,
+      {
+        validateStatus: function () {
+          // This ensures the promise is always resolved, preventing axios from throwing an error
+          return true;
+        },
+      },
+    );
+
+    if (enrichedCompany.status !== 200) {
+      return {};
+    }
+
+    return {
+      linkedinLinkUrl: `https://linkedin.com/` + enrichedCompany.data.handle,
+    };
+  }
+
+  async completeWithAi(content: string) {
+    fetch('https://openrouter.ai/api/v1/chat/completions', {
```

Comment:
@FelixMalfait Why using axios for one, and fetch for the other one ?
I rather use axios for both and use the Nest.JS implementation: https://docs.nestjs.com/techniques/http-module

Reply:
Good point! I blindly copied the OpenRouter docs without thinking